spec.md

```diff
@@ -179,10 +179,17 @@ as a string.
   `Integer` values in this range.
   - String encoding: Integer portion of the JSON Number per
     [RFC 7159, Section 6](https://tools.ietf.org/html/rfc7159#section-6)
-- `String` - Sequence of Unicode characters. "Control" characters in
-  the range U+0000-U+001F, except for U+000A LINE FEED and U+000D 
-  CARRIAGE RETURN, SHOULD NOT be included, since they have no agreed-on
-  meaning and predictably cause interoperability problems.
+- `String` - Sequence of allowable Unicode characters. The following characters
+  are disallowed:
+  - the "control characters" in the ranges U+0000-U+001F and U+007F-009F (both
```

Comment:
Either `U+0000-U+001F and U+007F-U+009F` or `U+0000-001F and U+007F-009F` 😉

Reply:
I had to look at the comment for an endless time to spot the fix but I did eventually …